src/components/shared/SamplesTable/SamplesTableAction.js

```diff
@@ -19,6 +19,7 @@ export const SamplesTableAction = ({ accessionCode, downloadableSamples }) => {
         data={{
           [accessionCode]: { all: true, total: downloadableSamples }
```

Comment:
The same thing applies here as above. Lets use the same function as described above to generate this structure.

Reply:
Please see my comment above, thank you David.